### PR TITLE
Json NEP6Wallet with basic tests

### PR DESCRIPTION
**Pull request by igormcoelho**
This may help you @shargon on https://github.com/neo-project/neo/pull/791

### CODE REVIEW

**Comment by igormcoelho** [comment]
@erikzhang I need a small help here... do you know why second test is giving null for ScryptParameters inside NEP6Wallet?

**Comment by igormcoelho** [comment]
Nevermind, all tests passed now!! \o/

File: neo/Wallets/NEP6/NEP6Wallet.cs [hunk 1]
```diff
@@ -49,6 +45,21 @@ public NEP6Wallet(string path, string name = null)
             }
         }
 
+        public NEP6Wallet(JObject wallet)
+        {
+            this.path = "";
+            LoadFromJson(wallet, out Scrypt, out accounts, out extra);
+        }
+
+        private void LoadFromJson(JObject wallet, out ScryptParameters Scrypt, out Dictionary<UInt160, NEP6Account> accounts, out JObject extra)
```
Comment (shargon) [inline neo/Wallets/NEP6/NEP6Wallet.cs]: `Script` better in lowercase

**Comment by igormcoelho** [comment]
@vncoelho json wallet, fine for you?

**Pull request merged**